docs/user-guide/federation/deployment.md

```diff
@@ -0,0 +1,104 @@
+---
+---
+
+This guide explains how to use Deployments in the Federation control plane.
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+This guide assumes that you have a running Kubernetes Cluster
+Federation installation. If not, then head over to the
+[federation admin guide](/docs/admin/federation/) to learn how to
+bring up a cluster federation (or have your cluster administrator do
+this for you). Other tutorials, for example
+[this one](https://github.com/kelseyhightower/kubernetes-cluster-federation)
+by Kelsey Hightower, are also available to help you.
+
+You are also expected to have a basic
+[working knowledge of Kubernetes](/docs/getting-started-guides/) in
+general and [Deployment](/docs/user-guide/deployment.md) in particular.
+
+## Overview
+
+Deployments in federation control plane (referred to as "Federated Deployments" in
+this guide) are very similar to the traditional [Kubernetes
+Deployment](/docs/user-guide/deployment.md), and provide the same functionality.
+Creating them in the federation control plane ensures that the desired number of
+replicas exist across the registered clusters.
+
+**Federated Deployment is currently in ALPHA, which means that the core functionality
```

Comment:
"Currently" can get out of date. Reference the specific version instead.

"As of Kubernetes version 1.5, Federated Deployment is an Alpha feature. The core functionality of Deployment is present, but some features (such as full rollout compatibility) are still in development."